modules/rostests/apitests/shlwapi/SHPropertyBag.cpp

```diff
@@ -619,28 +619,33 @@ static void SHPropertyBag_OnRegKey(void)
     pPropBag->Release();
 
     // Create as write-only IPropertyBag2
+    IPropertyBag2 *pPropBag2;
     hr = SHCreatePropertyBagOnRegKey(hKey, L"PropBagTest", STGM_WRITE,
-                                     IID_IPropertyBag2, (void **)&pPropBag);
+                                     IID_IPropertyBag2, (void **)&pPropBag2);
     ok_long(hr, S_OK);
 
     // Write UI4
+    PROPBAG2 propBag2 = {};
+    propBag2.dwType = PROPBAG2_TYPE_DATA;
+    propBag2.vt = VT_UI4;
+    propBag2.pstrName = const_cast<LPOLESTR>(L"Name1");
     VariantInit(&vari);
     V_VT(&vari) = VT_UI4;
     V_UI4(&vari) = 0xDEADFACE;
-    hr = pPropBag->Write(L"Name3", &vari);
+    hr = pPropBag2->Write(1, &propBag2, &vari);
     ok_long(hr, E_NOTIMPL);
     VariantClear(&vari);
 
     // Read UI4
     VariantInit(&vari);
     V_UI4(&vari) = 0xFEEDF00D;
-    hr = pPropBag->Read(L"Name3", &vari, NULL);
+    hr = pPropBag2->Read(1, &propBag2, NULL, &vari, NULL);
```

Comment:
According to
https://learn.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/platform-apis/aa768194(v=vs.85)
the last parameter `phrError` cannot be NULL.